internal/metapost.js

```diff
@@ -977,7 +977,7 @@ function isLeaf(block) {
 })();
 
 (function() {
-  function MetapostWriter() {
+  function MetapostWriter(opts) {
```

Comment:
Hmm... Actually no. I just copied the file from my local modification. I will remove it.